cypress/helpers/create-and-serve-paper-petition.ts

```diff
@@ -52,6 +77,55 @@ export function createAndServePaperPetition(
       cy.get('[data-testid="modal-confirm"]').click();
       cy.get('#done-viewing-paper-petition-receipt-button').click();
       cy.get('.usa-alert__text').should('have.text', 'Petition served to IRS.');
+
+      cy.get('[data-testid="docket-number-search-input"]').type(
+        `${docketNumber}`,
+      );
+      cy.get('.usa-search-submit-text').click();
+
+      cy.get('[data-test-case-attribute="status"]').should(
+        'have.text',
+        'General Docket - Not at Issue',
+      );
+
+      const expectedDocuments = [
+        { eventCode: 'P', index: 1, servedTo: 'R' },
+        { eventCode: 'APW', index: 2, servedTo: 'R' },
+        { eventCode: 'DISC', index: 3, servedTo: 'R' },
+        { eventCode: 'RQT', index: 4, servedTo: 'R' },
+        { eventCode: 'NOTR', index: 5, servedTo: 'P' },
+      ];
+
+      expectedDocuments.forEach(({ eventCode, index, servedTo }) => {
+        cy.get(
+          `[data-test-document-index="${index}"] > [data-test-document-meta="eventCode"]`,
+        ).should('have.text', eventCode);
+        cy.get(
+          `[data-test-document-index="${index}"] > [data-test-document-meta="servedPartiesCode"]`,
```

Comment:
For cypress I just wanted to ask about a couple of the different selectors I see. 
Why did you opt for data-test-document-index, data-test-document-meta as opposed to data-test-id?

Reply:
1.`data-testid` was already present on the row we were querying. And it felt like a daunting challenge to determine the `docketEntryId` for each docket entry, which is what it would have taken during the process of filing and serving the petition to the IRS with all of the attachments. I thought this was much more representative of how Tenille would test it and be more maintainable in the long run.
2. `data-testid` is not descriptive. Now when I read the test as a human, I have a sense of what I'm looking for. 
3. Other tests can now more easily check the row for a specific docket record index rather than having to know the `docketEntryId`.

Reply:
Those make sense. I think wanting more descriptive selectors is a good thing. My suggestion would be to make the value descriptive, not the html attribute descriptive to achieve that. I suggest that for two reasons:

1. consistency in selectors
2. Cypress specifically looks for `data-testid` (among others) when determining a good selector to use with the new record feature of cypress. 
![image](https://github.com/ustaxcourt/ef-cms/assets/81588260/cbaf8e9e-9cb7-4e43-a9ed-ba5e49b73221)
https://docs.cypress.io/api/cypress-api/selector-playground-api#Default-Selector-Priority

My suggestion would be to do something like data-testid=`document-index-{index}` as opposed to data-test-document-index=`${index}`

Reply:
Nice job!